CMakeLists.txt

```diff
@@ -68,6 +68,12 @@ add_custom_target(coveralls
   COMMENT "Send code coverage data to the coveralls.io service"
 )
 
+add_custom_target(test
```

Comment:
Let's add this just below the `luatest` target. And on second thought, the name should be something similar, like `luatest-no-coverage`.